### PR TITLE
Reorganize application code

### LINKED ISSUE #1005: Reorganize the API logic to make repo merger easier
### Is your feature request related to a specific problem?

Preparation for the repo merger

### Describe the solution you'd like

Nest modules where it makes sense to make the repo merger less messy

### PR DESCRIPTION
**Pull request by ThomasLaPiana**
# Purpose

Reorganize the code to make the repo merger easier

# Changes
* [x] nest the application code down to match the fidesctl layout

# Checklist
- [x] Update [`CHANGELOG.md`](https://github.com/ethyca/fidesops/blob/main/CHANGELOG.md) file
  - [x] Merge in main so the most recent `CHANGELOG.md` file is being appended to
  - [x] Add description within the `Unreleased` section in an appropriate category. Add a new category from the list at the top of the file if the needed one isn't already there.
  - [x] Add a link to this PR at the end of the description with the PR number as the text. example: [#1](https://github.com/ethyca/fidesops/pull/1)
- [x] Applicable documentation updated (guides, quickstart, postman collections, tutorial, fidesdemo, [database diagram](https://github.com/ethyca/fidesops/blob/main/docs/fidesops/docs/development/update_erd_diagram.md).
- If docs updated (select one):
  - [x] documentation complete, or draft/outline provided (tag docs-team to complete/review on this branch)
  - [ ] documentation issue created (tag docs-team to complete issue separately)
- [ ] Good unit test/integration test coverage
- [ ] This PR contains a DB migration. If checked, the reviewer should confirm with the author that the [down_revision correctly references the previous migration](https://ethyca.github.io/fidesops/development/contributing_details/#alembic-migrations) before merging
- [ ] The `Run Unsafe PR Checks` label has been applied, and checks have passed, if this PR touches any external services

# Ticket

Fixes #1005

### CODE REVIEW

**Comment by ThomasLaPiana** [comment]
@PSalant726 I'm assuming that the CodeQL warning is getting triggered because of the new Path?

**Comment by ThomasLaPiana** [comment]
documentation _should_ have gotten updated here with the sweeping find and replaces, I'll take a look at what changed to double check though

**Comment by sanders41** [comment]
One thing I found in testing that I haven't found a reason for yet. I am having with both `nox -s dev` and `docker-compose up`, when I try to login in from the admin-ui front end I get blocked. No log ever shows up on the server end as an attempted login. I compared the route the login page is using to the FastAPI docs route and everything looks correct. Maybe the CORS settings aren't getting applied?

**Comment by PSalant726** [comment]
> I'm assuming that the CodeQL warning is getting triggered because of the new Path?

Looks like a false positive because our variables contain the literal word "secret" ? My guess is that we dismissed a similar warning before the path changed, and now CodeQL is picking it up again.

The logging is not printing anything sensitive, so I think the warning can be dismissed. Someone with more context for this code should weigh in, just to be safe.

File: src/fidesops/ops/schemas/connection_configuration/connection_secrets_postgres.py.fb52500436032a279c9a6d942f00471c.tmp [hunk 1]
```diff
@@ -0,0 +1,24 @@
+from typing import List, Optional
```
Comment (sanders41) [inline src/fidesops/ops/schemas/connection_configuration/connection_secrets_postgres.py.fb52500436032a279c9a6d942f00471c.tmp:1]: Real file?

File: src/fidesops/ops/schemas/connection_configuration/connection_secrets_redshift.py.77c5f7672af2605bcc4e7cc6daa6eb67.tmp [hunk 2]
```diff
@@ -0,0 +1,23 @@
+from typing import List, Optional
```
Comment (sanders41) [inline src/fidesops/ops/schemas/connection_configuration/connection_secrets_redshift.py.77c5f7672af2605bcc4e7cc6daa6eb67.tmp:1]: Real file?

File: src/fidesops/ops/schemas/connection_configuration/connection_secrets_saas.py.ed63af19d9f256b37308ea42be206393.tmp [hunk 3]
```diff
@@ -0,0 +1,68 @@
+import abc
```
Comment (sanders41) [inline src/fidesops/ops/schemas/connection_configuration/connection_secrets_saas.py.ed63af19d9f256b37308ea42be206393.tmp:1]: Real file?

File: src/fidesops/ops/schemas/connection_configuration/connection_secrets_snowflake.py.05f3dd92bf04cb2ae63a08bc62915746.tmp [hunk 4]
```diff
@@ -0,0 +1,30 @@
+from typing import List, Optional
```
Comment (sanders41) [inline src/fidesops/ops/schemas/connection_configuration/connection_secrets_snowflake.py.05f3dd92bf04cb2ae63a08bc62915746.tmp:1]: Real file?

File: src/fidesops/ops/schemas/connection_configuration/connections_secrets_https.py.2fd7ff09dc87c6d90e22ec1618bf7dba.tmp [hunk 5]
```diff
@@ -0,0 +1,19 @@
+from typing import List
```
Comment (sanders41) [inline src/fidesops/ops/schemas/connection_configuration/connections_secrets_https.py.2fd7ff09dc87c6d90e22ec1618bf7dba.tmp:1]: Real file?

File: src/fidesops/ops/schemas/saas/strategy_configuration.py.aef003cd00155757a17dab128eb4461e.tmp [hunk 6]
```diff
@@ -0,0 +1,126 @@
+from enum import Enum
```
Comment (sanders41) [inline src/fidesops/ops/schemas/saas/strategy_configuration.py.aef003cd00155757a17dab128eb4461e.tmp:1]: Real file?

File: src/fidesops/ops/schemas/storage/storage.py.f967f57e435c2cbb2182dfe29b48043c.tmp [hunk 7]
```diff
@@ -0,0 +1,230 @@
+import logging
```
Comment (sanders41) [inline src/fidesops/ops/schemas/storage/storage.py.f967f57e435c2cbb2182dfe29b48043c.tmp:1]: Real file?

File: src/fidesops/ops/schemas/storage/storage_secrets_docs_only.py.a2fc43262f55fddaaf89d68b6464178a.tmp [hunk 8]
```diff
@@ -1,7 +1,7 @@
 from typing import Union
 
-from fidesops.schemas.base_class import NoValidationSchema
-from fidesops.schemas.storage.storage import StorageSecretsOnetrust, StorageSecretsS3
+from fidesops.ops.schemas.base_class import NoValidationSchema
```
Comment (sanders41) [inline src/fidesops/ops/schemas/storage/storage_secrets_docs_only.py.a2fc43262f55fddaaf89d68b6464178a.tmp:3]: Real file?

File: src/fidesops/ops/schemas/third_party/onetrust.py.63a7bde81bb019cc9f8851d685e70cf6.tmp [hunk 9]
```diff
@@ -0,0 +1,114 @@
+from enum import Enum
```
Comment (sanders41) [inline src/fidesops/ops/schemas/third_party/onetrust.py.63a7bde81bb019cc9f8851d685e70cf6.tmp:1]: Real file?

File: src/fidesops/ops/service/masking/strategy/format_preservation.py.ddc34b1ccfdc2a37c55cb893c9acae9d.tmp [hunk 10]
```diff
@@ -0,0 +1,11 @@
+from fidesops.ops.schemas.masking.masking_configuration import FormatPreservationConfig
```
Comment (sanders41) [inline src/fidesops/ops/service/masking/strategy/format_preservation.py.ddc34b1ccfdc2a37c55cb893c9acae9d.tmp:1]: Real file?

File: src/fidesops/ops/service/masking/strategy/masking_strategy.py.d5329929fe4d343efe0783f16ecd2ad7.tmp [hunk 11]
```diff
@@ -0,0 +1,42 @@
+# MR Note - It would be nice to enforce this at compile time
```
Comment (sanders41) [inline src/fidesops/ops/service/masking/strategy/masking_strategy.py.d5329929fe4d343efe0783f16ecd2ad7.tmp:1]: Real file?

File: src/fidesops/ops/service/pagination/pagination_strategy.py.e283a1a6aca0f4ffb543800d68e1d012.tmp [hunk 12]
```diff
@@ -0,0 +1,40 @@
+from __future__ import annotations
```
Comment (sanders41) [inline src/fidesops/ops/service/pagination/pagination_strategy.py.e283a1a6aca0f4ffb543800d68e1d012.tmp:1]: Real file?

File: src/fidesops/ops/service/storage/storage_authenticator_service.py.ee7ec8d03cceaa74a240905af062c2a6.tmp [hunk 13]
```diff
@@ -0,0 +1,60 @@
+import logging
```
Comment (sanders41) [inline src/fidesops/ops/service/storage/storage_authenticator_service.py.ee7ec8d03cceaa74a240905af062c2a6.tmp:1]: Real file?

File: src/fidesops/ops/util/encryption/hmac_encryption_scheme.py.b9d126a75d1079361d032501a4a4f618.tmp [hunk 14]
```diff
@@ -0,0 +1,62 @@
+import hashlib
```
Comment (sanders41) [inline src/fidesops/ops/util/encryption/hmac_encryption_scheme.py.b9d126a75d1079361d032501a4a4f618.tmp:1]: Real file?

File: src/fidesops/ops/util/encryption/secrets_util.py.c78f77783bb8ed6ba4d595fafe4abb32.tmp [hunk 15]
```diff
@@ -0,0 +1,75 @@
+import logging
```
Comment (sanders41) [inline src/fidesops/ops/util/encryption/secrets_util.py.c78f77783bb8ed6ba4d595fafe4abb32.tmp:1]: Real file?

File: src/fidesops/ops/util/storage_authenticator.py.561093629ca233945ec694fb6cc060a1.tmp [hunk 16]
```diff
@@ -0,0 +1,36 @@
+import boto3
```
Comment (sanders41) [inline src/fidesops/ops/util/storage_authenticator.py.561093629ca233945ec694fb6cc060a1.tmp:1]: Real file?

File: tests/ops/schemas/test_user_permission.py.2807084f66f944f2946e52a024651201.tmp [hunk 17]
```diff
@@ -0,0 +1,22 @@
+import pytest
```
Comment (sanders41) [inline tests/ops/schemas/test_user_permission.py.2807084f66f944f2946e52a024651201.tmp:1]: Real file?

File: tests/ops/util/encryption/test_hmac_encryption_scheme.py.489572ee3fc9884153f9619f1c4d75c9.tmp [hunk 18]
```diff
@@ -0,0 +1,29 @@
+from fidesops.ops.schemas.masking.masking_configuration import HmacMaskingConfiguration
```
Comment (sanders41) [inline tests/ops/util/encryption/test_hmac_encryption_scheme.py.489572ee3fc9884153f9619f1c4d75c9.tmp:1]: Real file?

File: tests/ops/test_helpers/cache_secrets_helper.py.b4d682a1bea8a757d49d14055d706429.tmp [hunk 19]
```diff
@@ -0,0 +1,23 @@
+from fidesops.ops.schemas.masking.masking_secrets import MaskingSecretCache
```
Comment (sanders41) [inline tests/ops/test_helpers/cache_secrets_helper.py.b4d682a1bea8a757d49d14055d706429.tmp:1]: Real file?

**Review by sanders41 — COMMENTED** [review]
There is one thing in the docs part that looks like it needs updating. Then every question after that is the same, are the `.tmp` files supposed to be there?

So it looks like a lot but the answer will be the same for all but one of the comments.

**Comment by ThomasLaPiana** [comment]
@sanders41 Aha, I know what happened with all of those tmp files. The find & replace took a toll on my computer (100+ files and automatic formatting) and VS code crashed, I'm assuming where those all came from, as they were in limbo waiting to be saved or something. Will delete them and update the docs page

**Comment by ThomasLaPiana** [comment]
@sanders41 I made some more thorough updates, would appreciate another review! 🙏

**Pull request merged**